package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "geostyler-sld-parser",
-  "version": "1.2.0",
+  "version": "1.3.1",
```

Comment:
Please reset to current version. Releases will be done separately.

Reply:
Sure, fixed now!